server_action_navigate/__manifest__.py

```diff
@@ -1,17 +1,18 @@
+# Migration 2022 - 14.0: Ashish Hirpara (https://ashish-hirpara.com)
 # Copyright (C) 2020 - Today: GRAP (http://www.grap.coop)
 # @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
     "name": "Server Actions - Navigate",
-    "version": "13.0.1.0.0",
-    "author": "GRAP, " "Odoo Community Association (OCA)",
+    "version": "14.0.1.0.0",
+    "author": "Ashish Hirpara, GRAP, Odoo Community Association (OCA)",
```

Comment:
your name shouldn't be here judging from the changes, listing yourself in the contributors is enough

Reply:
Done. And I will take care of this from now on.